### ISSUE TITLE
Distribution Not Loading - High

### ISSUE THREAD

**Issue by Happs80** [ReliefApplications/bms_front#1008]
Distribution marked as save for offline mode by another user seems to have marked it as offline for all users. So now opening the distribution, it won't load.
![Screenshot_20190725-073942](https://user-images.githubusercontent.com/31128624/61848799-9727f980-aeaf-11e9-9d66-4b8cbc1b4111.jpg)

**Comment by bamoure** [comment]
Offline is linked to the device not the user

**Comment by Happs80** [comment]
Yes, that's what i mean. Something is causing one user clicking 'offline mode' leading to other users seeing the same thing.
